test/integration/targets/prepare_http_tests/tasks/main.yml

```diff
@@ -1,7 +1,7 @@
 # The docker --link functionality gives us an ENV var we can key off of to see if we have access to
 # the httptester container
 - set_fact:
-    has_httptester: "{{ lookup('env', 'HTTPTESTER') != '' }}"
+    has_httptester: lookup('env', 'HTTPTESTER') != ''
```

Comment:
From the error its communicating with httpbin.org when it should be talking to httptester.